src/js/themes/base.js

```diff
@@ -1036,6 +1040,73 @@ export const generate = (baseSpacing = 24, scale = 6) => {
       // colors: [] || colors: ['graph-0', 'graph-1', 'graph-2', 'graph-3'],
       // extend: undefined,
     },
+    notification: {
+      container: {
+        // any box props
+        pad: { horizontal: 'small', vertical: 'xsmall' },
+        background: {
+          color: 'background-front',
+        },
+      },
+      toast: {
+        container: {
+          // any box props
+          elevation: 'medium',
+          round: 'xsmall',
+          width: 'medium',
+        },
+      },
+      iconContainer: {
+        // any box props
+        pad: { right: 'small' },
+      },
+      textContainer: {
+        // any box props
+        gap: 'medium',
+      },
+      title: {
+        // any text props
+        weight: 'bold',
+        color: {
+          light: 'black',
```

Comment:
wondering if these should be `text-strong` as opposed to hardcoded css color values